Sources/TuistKit/Commands/CleanCommand.swift

```diff
@@ -9,7 +10,12 @@ struct CleanCommand: ParsableCommand {
         )
     }
 
+    @Argument(help: "The cache categories to be cleaned. If no category is specified, the whole cache is cleaned.")
+    var cacheCategories: [CacheCategory] = CacheCategory.allCases
```

Comment:
does the help for this property describes what the individual values user can use and what they actually represent?

Reply:
yep, the values are automatically added by the library:
```
ARGUMENTS:
  <cache-categories>      The cache categories to be cleaned. If no category is
                          specified, the whole cache is cleaned. (default:
                          plugins, builds, tests, generatedAutomationProjects,
                          projectDescriptionHelpers, manifests)
```

I wouldn't go into the details of what each of them actually cleans

Reply:
While some are self-explanatory, for example `generatedAutomationProjects` could be a little confusing to the user. I think we should include the description of what the individual cache types do (probably both in the CLI and on the website).

Maybe even rewriting the autogenerated `default` bit might make sense here. I'd find `(default: whole tuist cache)` to be more readable than `(default: plugins, builds, tests, generatedAutomationProjects, projectDescriptionHelpers, manifests)`.